text/0000-component-level-contrib.md

```diff
@@ -0,0 +1,56 @@
+# Meta
+[meta]: #meta
+- Name: Guidelines for Component-level Contributions
+- Start Date: 2021-03-08
+- Author(s): @sclevine
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+
+# Motivation
+[motivation]: #motivation
+
+Our current contribution guidelines do not define a process for accepting component-level contributions, such as existing git repositories, that were developed outside of the project.
+While the lack of guidelines has not prevented components from being contributed (e.g., #123), establishing formal guidelines will ensure fair and orderly consideration of future proposals.
```

Comment:
```suggestion
While the lack of guidelines has not prevented components from being contributed (e.g., [#123](https://github.com/buildpacks/rfcs/pull/123)), establishing formal guidelines will ensure fair and orderly consideration of future proposals.
```
Special github linking for things like #123 don't appear to work in the body of a markdown document.